app/backend/aquifers/migrations/0009_auto_20190108_1925.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.4 on 2019-01-08 19:25
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aquifers', '0008_alter_sequence'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='aquifer',
+            name='mapping_year',
+            field=models.PositiveIntegerField(blank=True, help_text='Use the following format: <YYYY>', null=True, validators=[django.core.validators.MinValueValidator(1990), django.core.validators.MaxValueValidator(2019)], verbose_name='Date of Mapping'),
```

Comment:
I'm not too keen on those validators. Especially 2019 as an upper limit. We do have validation as a future story, so I would just drop those entirely right now. Also - is this part of the story?

This looks like a migration that was missed in another PR? I think we should just drop it from this one?